performance-timeline/po-disconnect.html

```diff
@@ -30,15 +30,16 @@
 
   async_test(function (t) {
     var observer = new PerformanceObserver(
-        t.step_func(function(entryList, obs) {
-          checkEntries(entryList.getEntries(),
-            [{ entryType: "mark", name: "mark1"}]);
-          t.done();
+        t.step_func(function (entryList, obs) {
+          assert_unreached("This callback must not be invoked");
```

Comment:
isn't it plausible that the callback is invoked between calling performance.mark("mark1") and observer.disconnect() (Although the original test did seem problematic)